bitsv/wallet.py

```diff
@@ -496,6 +601,9 @@ def __init__(self, wif=None):
         self.balance = 0
         self.unspents = []
         self.transactions = []
+        self.apis = apis
+        self.network = network
```

Comment:
I'd suggest moving all of this duplicate logic to a base class that can be shared with `PrivateKey`.

Reply:
Also, not really sure that I like using separate classes for the different networks. It would be really trivial to just do `my_key = PrivateKey(network='test')`

Reply:
The same thought has crossed my mind but I think for right this moment - I'd rather put this idea on the back-burner. But I think it's a good point. We could even just have PrivateKeyTestnet inherit from PrivateKey and override the 2 or 3 things that differ... or is that not aesthetic?